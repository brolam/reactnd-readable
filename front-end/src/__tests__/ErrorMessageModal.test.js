import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'
import ErrorMessageModal from '../components/ErrorMessageModal'

test('render without error', () => {
  ReactDOM.render(
    <ErrorMessageModal
      message={"An error happened, please try again."}
    />, document.createElement('div'));
})

test('last Snapshot', () => {
  const waitProcessModal = renderer.create(
    <ErrorMessageModal
      message={"An error happened, please try again."}
    />);
  let tree = waitProcessModal.toJSON();
  expect(tree).toMatchSnapshot();
})