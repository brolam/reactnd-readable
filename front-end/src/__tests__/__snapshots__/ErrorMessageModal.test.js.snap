// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`last Snapshot 1`] = `
<div
  className="modal-short modal-open"
>
  <div
    className="modal-short-dialog error-message-dialog"
  >
    <h1>
      An error happened, please try again.
    </h1>
  </div>
</div>
`;
